packages/office-addin-manifest/test/manifests/manifest.json

```diff
@@ -1,120 +1,82 @@
 {
   "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
-  "id": "df2fa15f-332a-47aa-87c2-e30c20686eee",
-  "manifestVersion": "m365DevPreview",
+  "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
   "version": "1.0.0",
+  "manifestVersion": "m365DevPreview",
   "name": {
-    "short": "My Office Add-in",
-    "full": "My Office Add-in"
+    "short": "Contoso Task Pane Add-in",
+    "full": "Contoso Task Pane Add-in"
   },
   "description": {
     "short": "A template to get started.",
-    "full": "A template to get started."
+    "full": "This is the template to get started."
   },
-  "icons": {
-    "outline": "test/assets/icon-32.png",
-    "color": "test/assets/icon-32.png"
-  },
-  "accentColor": "#FFFFFF",
   "developer": {
     "name": "Contoso",
     "websiteUrl": "https://www.contoso.com",
     "privacyUrl": "https://www.contoso.com/privacy",
-    "termsOfUseUrl": "https://www.contoso.com/terms"
+    "termsOfUseUrl": "https://www.contoso.com/servicesagreement"
+  },
+  "icons": {
+    "outline": "test/assets/icon-32.png",
+    "color": "test/assets/icon-32.png"
   },
+  "accentColor": "#230201",
   "localizationInfo": {
-    "defaultLanguageTag": "en-us"
+    "defaultLanguageTag": "en-us",
+    "additionalLanguages": [
```

Comment:
I think we need to start with this as an empty array since we don't have language files as part of our template (or do we?).